src/api/test/functional/attributes_test.rb

```diff
@@ -131,6 +126,10 @@ def test_create_type
     assert_response :success
     get "/attribute/TEST/Dummy/_meta"
     assert_response :success
+    login_Iggy
+    delete "/attribute/TEST/Dummy/_meta"
+    assert_response 403
+    login_adrian
```

Comment:
please don't move api tests to rspec as long we don't validate the API requests and responses via schemas there.